storybookjs/storybook
# Telemetry: Count onboarding stories

N/A

## What I did

Add onboarding story count per  #22558

## How to test

See attached tests, or run a sandbox with `STORYBOOK_TELEMETRY_DEBUG=1` and add a `Warning` story to the Button.
